pysatNASA/instruments/methods/general.py

```diff
@@ -0,0 +1,50 @@
+"""General methods for NASA instruments."""
+
+import warnings
+
+import pysat
+
+
+def init(self, module, name):
+    """Initialize the Instrument object with instrument specific values.
+
+    Parameters
+    -----------
+    module : python module
```

Comment:
```suggestion
    module : module
```

To keep type standards used in pysat.